projects/cdk/tokens/ensure-base-href.ts

```diff
@@ -5,11 +5,13 @@ export const TUI_ENSURE_BASE_HREF: InjectionToken<string> = new InjectionToken<s
     `[TUI_ENSURE_BASE_HREF]`,
     {
         factory: () => {
+            const baseHref = inject(DOCUMENT).getElementsByTagName(`base`)?.[0]?.href;
+
             const link = inject(DOCUMENT).createElement(`a`);
```

Comment:
Should we do early exit if `baseHref` exists ?

Reply:
fixed